### PR TITLE
irqbalance: init at 1.1.0

### PR DESCRIPTION
**Pull request by marcinfalkiewicz**
A daemon, which spreads hardware interrupts across multiple cpus (instead of clogging all of them on CPU0).

### CODE REVIEW

File: nixos/modules/services/hardware/irqbalance.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  cfg = config.services.irqbalance;
+
+in
+{
+
+  ###### interface
+  options = {
+    services.irqbalance = {
+      enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/hardware/irqbalance.nix]: Please use `mkEnableOption`.

**Comment by dwe11er** [comment]
@jagajaga I've changed mkOption to mkEnableOption.

**Comment by jagajaga** [comment]
Please squash commits into one.

File: pkgs/os-specific/linux/irqbalance/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchgit, autoconf, automake, libtool, pkgconfig}:
+
+stdenv.mkDerivation rec {
+  name = "irqbalance-1.1.0";
+
+  src = fetchgit {
```
Comment (jagajaga) [inline pkgs/os-specific/linux/irqbalance/default.nix]: Please use `fetchFromGitHub`.

**Comment by dwe11er** [comment]
@jagajaga Should be fine now. Thanks for guidance.

**Comment by jagajaga** [comment]
Oh, I'm sorry, but can you rename commit to follow our guides in `CONTRIBUTING.md`?
> `ircbalance: init at 1.1.0`

**Pull request merged**